plugins/out_s3/s3_win32_compat.h

```diff
@@ -0,0 +1,64 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
```

Comment:
Once the PR is ready to be merged, it should be good to move the compat logic to the formal `include/fluent-bit/flb_compat.h`

Reply:
Now, I moved them into _flb_compat.h_.
https://github.com/fluent/fluent-bit/pull/8837/commits/db9eb924edc47b131210c738ed5fda0301956e47